docs/en/core-services/Caching.md

```diff
@@ -333,32 +379,32 @@ $ ./bin/alluxio fs free ${PATH_TO_UNUSED_DATA}
 
 This will remove the data at the given path from Alluxio storage. The data is still accessible if
 it is persisted to a UFS. For more information refer to the
-[command line interface documentation]({{ '/en/operation/User-CLI.html' | relativize_url }}#free)
+[command line interface documentation]({{ '/en/operation/User-CLI.html#free' | relativize_url }})
```

Comment:
I think the hashtag part should go together with the URL, as how we do it in most other docs. This change is tested.